addons/maintenance/migrations/13.0.1.0/pre-migration.py

```diff
@@ -3,6 +3,19 @@
 from openupgradelib import openupgrade
 
 
+_column_renames = {
+    'maintenance_request': [
+        ('active', None),
+    ],
+}
+
+
+def rename_column_active(env):
+    if openupgrade.column_exists(
+            env.cr, "maintenance_request", "active"):
+        openupgrade.rename_columns(env.cr, _column_renames)
```

Comment:
nhét luôn đoạn `openupgrade.rename_columns(env.cr, _column_renames)` xuống dưới hàm `migrate` luôn cho gọn, tạo ra 1 hàm chỉ để rename cột `active` nhưng lại sử dụng biến toàn cục `_column_renames` mà có thể chứa nhiều cột thì ko hợp lý.

Reply:
hoặc bỏ cái biến toàn cục đi thì hơn, vì trong trường hợp này cũng cần check xem có tồn tại cột `active` hay ko thì cũng nên tách 1 hàm riêng
```suggestion
def rename_column_active(env):
    if openupgrade.column_exists(
            env.cr, "maintenance_request", "active"):
        openupgrade.rename_columns(env.cr, {
            'maintenance_request': [
                ('active', None),
            ],
        })
```

Reply:
done a nhé.